CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 Rack compatible HTTP router for Ruby
 
+## [Unreleased]
+
+### Changed
+
+- [Sven Schwyn] Convert dash to underscore in URL helper names (#280)
```

Comment:
```suggestion
- [Sven Schwyn] Convert -, +, ~, and . to underscore in URL helper names (#280)
```

Reply:
Done... here and equally in the commit message.